onnxruntime/core/providers/openvino/backend_manager.cc

```diff
@@ -24,15 +24,6 @@ BackendManager::BackendManager(const GlobalContext& global_context,
   global_context_ = global_context;
 
   auto prec_str = GetGlobalContext().precision_str;
-  if (prec_str == "FP32") {
```

Comment:
Should we keep this around till we extend the scope of device_precision separation to other devices like GPU?